src/index.ts

```diff
@@ -198,55 +201,56 @@ class CruxPayPeer extends EventEmitter {
         return (this._payIDClaim as PayIDClaim);
     }
 
-    public registerCruxID = async (virtualAddress: string): Promise<void> => {
-        this._setPayIDClaim(new PayIDClaim({virtualAddress}, { getEncryptionKey: this._getEncryptionKey }));
-        // await this._payIDClaim.setPasscode(passcode)
-        await (this._payIDClaim as PayIDClaim).encrypt();
-        this._storage.setJSON("payIDClaim", (this._payIDClaim as PayIDClaim).toJSON());
-        await this._restoreIdentity();
-    }
+    public registerCruxID = async (cruxIDSubdomain: string): Promise<void> => {/**/};
```

Comment:
- [ ] validate and proceed if this can be entirely removed from the base class `CruxPayPeer`.